ramls/holdings_update_ownership.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
@RomanChernetskyi  I think these schemas should be data-import-raml-storage. Is it correct?

Reply:
nope, this functionality is not related to data-import. It is inventory domain

Reply:
@KaterynaSenchenko Thank you for the explanation.